CosmosContracts/juno
# [ictest]: LocalJuno

Closes #657 

WIP - Builds off 47-upgrade branch as we use ictest v7 there

# Files of importance
- interchaintest/localjuno/*

# Completed Features
- Config use ibc-path as a unique id to link chains together 

# TODO
- move to ictest natively
- Mess with SQL indexer built in (make sure to do an array of common modules to index)
- goals in readme
- remove 2 path ibc limit
- remove current docker script things
- have a simple localjuno with 0 config (maybe allow for CLI/OS ARGS?)
- Build & push on tags/releases
- Add built-in rest api: make relay connection during runtime (between chains, or contracts)
